src/toolkits/activity_classification/activity_classifier.cpp

```diff
@@ -524,6 +525,11 @@ void activity_classifier::import_from_custom_model(
   flex_dict pred_model = variant_get_value<flex_dict>(it->second);
   model_data.erase(it);
 
+  //generate a random seed for the model
```

Comment:
Why is a random seed required for evaluation?

Reply:
it somehow need to sample the time series data with a random seed.